cypress-tests/cypress/e2e/editor/inspectorHappypath.cy.js

```diff
@@ -170,4 +170,10 @@ describe("Editor- Inspector", () => {
     cy.notVisible(commonWidgetSelector.draggableWidget("button1"));
     cy.apiDeleteApp();
   });
+  it("should verify deletion of component from inspector", () => {
+    cy.dragAndDropWidget("button", 500, 500);
+    cy.get(commonWidgetSelector.sidebarinspector).click();
+    deleteComponentFromInspector("button1");
+
```

Comment:
- I think we need to add assertion to verify the deletion of component and the toast message, after the deletion.
- Let's create one more version and verify that the components is not exist on canvas.